config/environments/development.rb

```diff
@@ -15,6 +15,14 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.smtp_settings = {
+    address:              'smtp.gmail.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.